make/Makefile.container.mk

```diff
@@ -17,7 +17,11 @@
 	    echo "You do not have operator-sdk installed in your PATH. Will use the one found here: ${OUTDIR}/operator-sdk-install/operator-sdk" ;\
 	  else \
 	    echo "You do not have operator-sdk installed in your PATH. The binary will be downloaded to ${OUTDIR}/operator-sdk-install/operator-sdk" ;\
-	    curl -L https://github.com/operator-framework/operator-sdk/releases/download/v0.10.0/operator-sdk-v0.10.0-x86_64-linux-gnu > "${OUTDIR}/operator-sdk-install/operator-sdk" ;\
+	    if [ "${GOARCH}" == "arm64" ]; then \
+		curl -L https://github.com/BlackMix/operator-sdk-arm64/releases/download/v0.16.0/operator-sdk-v0.16.0-27-g3687cc1a-dirty-aarch64-linux-gnu > "${OUTDIR}/operator-sdk-install/operator-sdk" ;\
+	    else \
+		curl -L https://github.com/operator-framework/operator-sdk/releases/download/v0.10.0/operator-sdk-v0.10.0-x86_64-linux-gnu > "${OUTDIR}/operator-sdk-install/operator-sdk" ;\
```

Comment:
@jmazzitelli is there any impact if we move to latest operator-sdk v0.16.0?
I'm curious why we've been stuck at v0.10.0.

Reply:
we can move to v0.16.0. No reason other than we are not keeping pace with the sdk releases.

Reply:
FYI: kiali just moved over to 0.16.0 in master via another PR: https://github.com/kiali/kiali/pull/2587

Reply:
Thanks! :))